### ISSUE TITLE
Cities referring to multiple geonames entries

### ISSUE THREAD

**Issue by alexprengere** [opentraveldata/opentraveldata#178]
There are many cities that link to multiple GeoNames entries (about 175 counting only those with pageranks). For example `AMS`:

```
AMS-A-6296680^AMS^52.3103^4.76028^AMS^
AMS-C-2759794^AMS^52.37403^4.88969^AMS^    <- Amsterdam: https://www.geonames.org/2759794
AMS-C-10173839^AMS^52.29843^4.79094^AMS^   <- Schiphol:  https://www.geonames.org/10173839
```

This is not a big issue, but I believe we could do some cleanup. One consequence of this is that the `city_code_list` and `city_name_list` do not have the same length. For `AMS` airport: 

```
city_code_list      ['AMS']
city_name_list      ['Amsterdam', 'Schiphol']
```

I don't think there is an automated way to "fix" this, or even if there is something to fix, but we could have a look at the optd_por_best_known_so_far. Perhaps @denisarnaud has more insights 😉

**Comment by denisarnaud** [comment]
Yes, absolutely, that is not a bug, rather a feature (and I struggled so that OPTD could support that)!

**Comment by denisarnaud** [comment]
So, either we migrate the city_xxx_list to some better data structure, or we allow city_code_list to have duplicated codes. That latter option may be easier to implement, I don't know for sure.
